scripts/setup.sh

```diff
@@ -188,24 +188,19 @@ step_6(){
 }
```

Comment:
patch cluster policy to tolerate the taint (add the oc patch command)

Reply:
added: to config yaml